lib/logstash/inputs/kafka.rb

```diff
@@ -155,6 +158,7 @@ def run(logstash_queue)
 
   public
   def stop
+    @kafka_client_queue.push(LogStash::SHUTDOWN)
```

Comment:
I'm not so sure we should repurpose the LogStash::SHUTDOWN object here. I could go either way on this, but this is more of a kafka shutdown. Maybe it should be some other constant? Anyway, if you feel like using the code as is go for it, but do think about it.